tasktiger/constants.py

```diff
@@ -0,0 +1,11 @@
+# constants pertaining to Redis keys
+REDIS_PREFIX = "t"
+TASK = "task"
+EXECUTIONS = "executions"
+EXECUTIONS_COUNT = "executions_count"
+
+
+QUEUED = "queued"
```

Comment:
These are already defined in `tasktiger._internal`.
What's more, these definitions aren't even used.
We should get rid of them.